src/XrdCeph/XrdCephOss.cc

```diff
@@ -128,19 +128,26 @@ int XrdCephOss::Configure(const char *configfn, XrdSysError &Eroute) {
        if (!strncmp(var, "ceph.namelib", 12)) {
          var = Config.GetWord();
          if (var) {
+           std::string libname = var;
            // Warn in case parameters were givne
            char parms[1040];
+           bool hasParms{false};
            if (!Config.GetRest(parms, sizeof(parms)) || parms[0]) {
-             Eroute.Emsg("Config", "namelib parameters will be ignored");
+              hasParms = true;
            }
            // Load name lib
-           XrdOucN2NLoader n2nLoader(&Eroute,configfn,NULL,NULL,NULL);
-           g_namelib = n2nLoader.Load(var, XrdVERSIONINFOVAR(XrdOssGetStorageSystem), NULL);
+           if (hasParms) {
+             XrdOucN2NLoader  n2nLoader(&Eroute,configfn,parms,NULL,NULL);
+             g_namelib = n2nLoader.Load(libname.c_str(), XrdVERSIONINFOVAR(XrdOssGetStorageSystem), NULL);
+           } else {
```

Comment:
in the else clause, should it be libname.c or var?

Reply:
In the latest commit, I've simplified this line, so libname is used consistently.

Reply:
all good, the logging will be good to check for weird behaviour as well